enterprise/internal/campaigns/store.go

```diff
@@ -2062,6 +2062,11 @@ type CountPatchesOpts struct {
 	// _not_ associated with a successfully completed ChangesetJob (meaning
 	// that a Changeset on the codehost was created) for the given Campaign.
 	OnlyUnpublishedInCampaign int64
+
+	// If this is set to a Campaign ID only the Patches are returned that are
+	// _not_ associated with any ChangesetJob (meaning
+	// that a Changeset on the codehost was created) for the given Campaign.
+	OnlyNeverPublishedInCampaign int64
```

Comment:
I think this is much clearer for all of our uses cases of the `*Store`.

```suggestion
	OnlyWithoutChangesetJobInCampaign int64
```